### ISSUE TITLE
Fork stochastic_physics into the NCAR Github account

### ISSUE THREAD

**Issue by gustavo-marques** [ESCOMP/MOM_interface#175]
We must fork [stochastic_physics](https://github.com/NOAA-PSL/stochastic_physics) into the NCAR Github account and make sure the changes from @iangrooms are merged.

**Comment by iangrooms** [comment]
I plan to submit a PR to the main NOAA repo from my fork soon - maybe a week or two. Can you wait that long?

**Comment by gustavo-marques** [comment]
Yes, we can wait. I discussed this with @mnlevy1981, and we agreed that it's okay to be using your fork as long as stochastic_physics is not enabled by default.

**Comment by iangrooms** [comment]
OK. It's on my to-do list.

**Comment by alperaltuntas** [comment]
Do we have an update on this issue? @gustavo-marques @iangrooms

**Comment by iangrooms** [comment]
ESCOMP now has a fork of the repository: ESCOMP/stochastic_physics. The `ocn_skeb` branch is up to date with my own, which is (I think) what MOM_interface is currently pointing to. So you could change MOM_interface so that it points to the ESCOMP fork instead of my own.

There will be a little extra work needed to merge with the main NOAA-PSL version of the repository, but @pjpegion will let me know when he's ready for that.

**Comment by mnlevy1981** [comment]
> ESCOMP now has a fork of the repository: ESCOMP/stochastic_physics. The `ocn_skeb` branch is up to date with my own, which is (I think) what MOM_interface is currently pointing to.

`MOM_interface` is [pointing to `ESCOMP/stochastic_physics`](https://github.com/ESCOMP/MOM_interface/blob/main/.gitmodules#L8-L13):

```

[submodule "stochastic_physics"]
  path = externals/stochastic_physics
  url = https://github.com/ESCOMP/stochastic_physics.git
  fxDONOTUSEurl = https://github.com/ESCOMP/stochastic_physics.git
    fxtag = ocn_skeb_240807
    fxrequired = AlwaysRequired
```

I think that was the goal of this issue, so I'm going to close it. (This was accomplished in #193)
